modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/persistence/ExtendedEventDto.java

```diff
@@ -64,44 +60,207 @@
   @Column(name = "organization", length = 128)
   private String organization;
 
+  /** Capture agent id */
+  @Column(name = "capture_agent_id", length = 128)
+  private String captureAgentId;
+
+  /** recording start date */
+  @Column(name = "start_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date startDate;
+
+  /** recording end date */
+  @Column(name = "end_date")
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date endDate;
+
+  /** source */
+  @Column(name = "source")
+  private String source;
+
+  /** recording state */
+  @Column(name = "recording_state")
+  private String recordingState;
+
+  /** recording last heard */
+  @Column(name = "recording_last_heard")
+  private Long recordingLastHeard;
+
+  /** review status */
+  @Column(name = "review_status")
+  private String reviewStatus;
```

Comment:
In the SQL script, this is defined as
```sql
review_status VARCHAR(128),
```
while the `review_status` above which in here has the same definition, has the following SQL definition:
```sql
review_status VARCHAR(128),
```
The definition in code and in SQL should match whenever possible